ctapipe/image/leakage.py

```diff
@@ -0,0 +1,53 @@
+"""
+Leakage calculation
+"""
+
+import numpy as np
+__all__ = ['leakage']
+
+def leakage(geom, image):
+    '''
+    Calculating the leakage-values for a given image.
+    Image must be cleaned for example with tailcuts_clean.
+
+    Parameters
+    ----------
+        geom: `ctapipe.instrument.CameraGeometry`
+            Camera geometry information
+        image: array
+            pixel values
+
+    Returns
+    -------
+        leakage_border_pixel1:   float
+            Number of shower-pixel on the border divided by all shower-pixel
+        leakage_border_pixel2:   float
+            Number of shower-pixel in the second row of the border divided by all shower-pixel
+        leakage_border_photon1:   float
+            Number of photons in the border-pixel divided by all photons
+        leakage_border_photon2:   float
+            Number of photons in the second row of the border-pixel divided by all photons
+    '''
+    max_value = max(np.sum(geom.neighbor_matrix, axis=0))
+    leakage_border_pixel1 = 0
+    leakage_border_pixel2 = 0
+    leakage_border_photon1 = 0
+    leakage_border_photon2 = 0
+    size = 0
+    pixel_count = 0
+    for i in range(len(image)):
```

Comment:
Maybe its faster if you only iterate over the non-zero pixel? I don't know. Shouldn't make a huge difference.